Makefile

```diff
@@ -1,3 +1,9 @@
+# generate version number from the latest git tag; if the topmost commit is
+# tagged (at release time), just use that; if there are commits after that,
+# append a ".x" suffix to indicate that's a development snapshot
+# if there is no tag (yet), use 0 as version
+RELEASEVER=$(shell (git describe || echo 0) | sed 's/-[0-9]\+-g.*/.x/')
```

Comment:
I think this needs to be a bit more complex (or just don't build without a proper version tag). We should be following semver.org rules for versions.

Reply:
Can you please expand what you mean with "more complex"? This doesn't make many assumptions about how release versions look like, other than that they have to start with a number (otherwise rpm complains).

I added the fallback to "0" for the current case where we dont' yet have any release (chicken and egg problem).